### PR TITLE
Display a message if user tries to set freq on linux

### LINKED ISSUE #682: Why can't I2C baudrate/frequency be set in Python?
For all microcontrollers supported by this project, there are lines commented out that state that the I2C frequency cannot be set in Python and is therefore ignored. because these print statements are commented out, the end user (me) has no idea why the API parameter isn't working.
https://github.com/adafruit/Adafruit_Blinka/blob/main/src/adafruit_blinka/microcontroller/generic_linux/i2c.py#L23-L24

I'm using a Raspberry Pi Zero W with the Adafruit motor control bonnet to control a stepper motor which is turning slower than I like. The Adafruit MotorKit documentation suggests updating the frequency to get more speed.
https://docs.circuitpython.org/projects/motorkit/en/latest/api.html#adafruit_motorkit.MotorKit

I've already changed the `config.txt` setting to update the I2C speed from 100khz to 400khz, but my understanding is that the code also needs to use it. I've confirmed that it's now 400khz.
https://learn.adafruit.com/raspberry-pi-i2c-clock-stretching-fixes/change-the-clock-speed

Three questions:

1. Why is this setting ignored, or, why can't it be changed in Python?
2. Can these lines be uncommented and changed to throw a Warning instead of ignoring the parameter silently?
3. If I can't changed the speed in Python, what should I do instead?

### PR DESCRIPTION
**Pull request by makermelissa**
Fixes #682. Also improves issue template messages.

### CODE REVIEW

File: src/adafruit_blinka/microcontroller/generic_linux/i2c.py [hunk 1]
```diff
@@ -20,8 +24,8 @@ def __init__(self, bus_num, mode=MASTER, baudrate=None):
             raise NotImplementedError("Only I2C Master supported!")
         _mode = self.MASTER
 
-        # if baudrate != None:
-        #    print("I2C frequency is not settable in python, ignoring!")
+        if baudrate is not None:
+            print("I2C frequency is not settable in python, ignoring!")
```
Comment (tannewt) [inline src/adafruit_blinka/microcontroller/generic_linux/i2c.py:28]: [`warnings`](https://docs.python.org/3/library/warnings.html) is probably the best way to do it so folks can turn it off.

**Review by tannewt — CHANGES_REQUESTED** [review]
One suggestion instead of print.

File: src/adafruit_blinka/microcontroller/generic_linux/i2c.py (hunk 1 above)
Comment (makermelissa) [reply src/adafruit_blinka/microcontroller/generic_linux/i2c.py:28]: Oh, I didn't know about this. Good idea.

**Review by tannewt — APPROVED** [review]
Thanks for the quick update!

**Pull request merged**